app/main/routes_zseries_api.py

```diff
@@ -298,12 +300,12 @@ def create_session(token, body):
     session_guid = uuid.uuid4().hex[:32]
     session_id = increment_session_id(uid)
     
-    active_brew_sessions[uid].guid = session_guid
+    active_brew_sessions[uid].session = session_guid
     active_brew_sessions[uid].id = session_id
     active_brew_sessions[uid].created_at = datetime.utcnow().isoformat()
     active_brew_sessions[uid].name = recipe.name if recipe else body['Name']
     active_brew_sessions[uid].type = body['SessionType']
-    active_brew_sessions[uid].filepath = Path(BREW_ACTIVE_PATH).joinpath('{0}#{1}#{2}#{3}#{4}.json'.format(datetime.now().strftime('%Y%m%d_%H%M%S'), uid, active_brew_sessions[uid].guid, active_brew_sessions[uid].name.replace(' ', '_'), active_brew_sessions[uid].type))
+    active_brew_sessions[uid].filepath = Path(BREW_ACTIVE_PATH).joinpath('{0}#{1}#{2}#{3}#{4}.json'.format(datetime.now().strftime('%Y%m%d_%H%M%S'), uid, active_brew_sessions[uid].session, active_brew_sessions[uid].name.replace(' ', '_'), active_brew_sessions[uid].type))
```

Comment:
though it doesn't seem like the Z uses this field I noticed it wasn't being populated correctly into the response